modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-partner-portal/extra/remote-app/src/routes/MDFRequestList/utils/getMDFListColumns.tsx

```diff
@@ -74,6 +81,32 @@ export default function getMDFListColumns(
 						}`
 					),
 			},
+			{
+				icon: 'trash',
+				key: 'delete',
+				label: ' Delete',
+				onClick: async () => {
+					if (row[MDFColumnKey.STATUS] === Status.DRAFT.name) {
+						await deleteMDFRequest(
+							ResourceName.MDF_REQUEST_DXP,
+							Number(row[MDFColumnKey.ID]) as number
+						);
+
+						Liferay.Util.openToast({
+							message: 'MDF Request Deleted successfully',
+							type: 'success',
+						});
+
+						mutate(mutated);
+					} else {
+						Liferay.Util.openToast({
+							message:
+								'You cannot delete the MDF Request in this status',
+							type: 'danger',
+						});
+					}
+				},
+			},
```

Comment:
@DiogoNo, eh melhor nao aparecer a opcao caso nao tenha permissao

Reply:
coloca numa condicional separada pra aparecer apenas se for Draft. O tratamento do erro e ser apenas se a requisicao der errado.